easybuild/easyconfigs/s/smooth-topk/smooth-topk-1.0-20210817-foss-2021a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = "PythonBundle"
+
+name = "smooth-topk"
+version = "1.0-20210817"
+local_commit = "12c1645f187e2fa0c05f47bf1fe48864d4bd2707"
+
+homepage = "https://github.com/oval-group/smooth-topk"
+description = "Smooth Loss Functions for Deep Top-k Classification"
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('tqdm', '4.61.2'),
+    ('PyTorch', '1.10.0'),
+    ('SciPy-bundle', '2021.05'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('future', '0.18.2', {
```

Comment:
this same version of `future` is already included with `Python` 3.9.5, so this extension can be removed